libshaderc_spvc/src/spvcir_pass.cc

```diff
@@ -83,28 +121,57 @@ void SpvcIrPass::GenerateSpirvCrossIR(Instruction *inst) {
     }
 
     case SpvOpCapability: {
-      auto cap = inst->GetSingleWordOperand(0u);
+      auto cap = inst->GetSingleWordInOperand(0u);
       assert(cap != spv::CapabilityKernel &&
              "Kernel capability not supported.");
       ir_->declared_capabilities.push_back(static_cast<spv::Capability>(cap));
       break;
     }
 
+    // opcode: 10
     case SpvOpExtension: {
       const std::string extension_name =
           reinterpret_cast<const char *>(inst->GetInOperand(0u).words.data());
       ir_->declared_extensions.push_back(std::move(extension_name));
       break;
     }
 
-    case SpvOpMemoryModel: {
-      ir_->addressing_model =
-          static_cast<spv::AddressingModel>(inst->GetSingleWordInOperand(0u));
-      ir_->memory_model =
-          static_cast<spv::MemoryModel>(inst->GetSingleWordInOperand(1u));
+    // opcode:11
+    case SpvOpExtInstImport: {
+      uint32_t id = inst->result_id();
+      std::string ext(
+          reinterpret_cast<const char *>(inst->GetInOperand(0u).words.data()));
+      if (ext == "GLSL.std.450")
+        set<spirv_cross::SPIRExtension>(id, spirv_cross::SPIRExtension::GLSL);
+      else if (ext == "DebugInfo")
+        set<spirv_cross::SPIRExtension>(
+            id, spirv_cross::SPIRExtension::SPV_debug_info);
+      else if (ext == "SPV_AMD_shader_ballot")
```

Comment:
It's unfortunate that SPIRV-Cross mixes the concepts of extended instruction set and extension.  But that's how it is, so this code must match that modeling.